pkg/gameservers/controller.go

```diff
@@ -288,11 +285,6 @@ func (c *Controller) creationValidationHandler(review admissionv1.AdmissionRevie
 	c.loggerForGameServer(gs).WithField("review", review).Debug("creationValidationHandler")
 
 	causes, ok := gs.Validate()
-	// Merge product-specific validation - we handle it here to avoid introducing cloudproduct to the api packages.
```

Comment:
Bye bye 👋🏻 😄

Reply:
Yeah, I'm happy with how this turned out, it was making me grumbly to have action at such a distance!